newscoop/library/Newscoop/Entity/Repository/ArticleRepository.php

```diff
@@ -165,11 +165,17 @@ public function getArticlesByCriteria($articleSearchCriteria, $ids = array(), $o
 
         $queryBuilder = $em->getRepository('Newscoop\Entity\Article')
             ->createQueryBuilder('a')
-            ->select('a, FIELD(a.number, :ids) as HIDDEN field')
+            ->select('a, FIELD(a.number, :ids) as HIDDEN field', 'l', 'u', 'ap', 'p', 'aa', 'au', 't')
             ->andWhere('a.number IN (:ids)')
-            ->leftJoin('a.publication', 'p')
             ->leftJoin('a.issue', 'i')
             ->leftJoin('a.section', 's')
+            ->leftJoin('a.packages', 'p')
```

Comment:
all those relations are anyway required by api on every call - and for now they were loaded with lazy load (sometimes creating 4-5x more queries than requested records). This will fix also issue for broken relation - with implict leftJoin it will just ignire broken relations - with lazy load it throws EntityNotFound exception and break whole api response.